src/modules/QtQuick/Text.js

```diff
@@ -127,6 +125,7 @@ QmlWeb.registerQmlType({
   }
   Component$onCompleted() {
     this.$updateImplicit();
+    this.$onWrapModeChanged(this.wrapMode);
```

Comment:
Note: this now calls `$updateImplicit()` two times. Perhaps something could be done with that?
If the `$onWrapModeChanged` always calls `$updateImplicit`, perhaps the first line could be removed (replaced with a comment that `$updateImplicit()` needs to be called, but that it's already called by `$onWrapModeChanged`).